tests/unit_tests/training/test_nvrx_straggler.py

```
@@ -280,41 +280,66 @@ def dummy_func():
         assert len(call_args) == 2
         wrapper_obj, method_name = call_args
         assert method_name == "train_step"
 
-        # Verify the wrapper object has the train_step method that wraps our dummy_func
+        # Verify the wrapper object has the train_step method
         assert hasattr(wrapper_obj, "train_step")
-        assert wrapper_obj.train_step is dummy_func
 
         mock_straggler_module.Detector.wrap_callables.assert_called_once()
 
+    def test_wrap_train_step_function_callable(self, manager, mock_straggler_module):
```

Ruff (0.14.14)

Unused method argument: `mock_straggler_module`

(ARG002)